sda-commons-server-testing/src/main/java/org/sdase/commons/server/testing/DropwizardRuleHelper.java

```diff
@@ -28,6 +28,14 @@
  *   - type: http
  *     port: 0
  * }</pre>
+ *     Instead of specifying connectors in a config file create random ports programmatically:
```

Comment:
Maybe not as strong: "Instead of ... you can create random ports programatically:"